src/index.tsx

```diff
@@ -66,12 +66,13 @@ const queryClient = new QueryClient()
 
 const AppWithProviders = () => {
   const migrated = useMigrations()
+
   return migrated ? (
     <WalletManagerProvider walletManager={walletManager}>
       <ErrorBoundary>
         <Provider store={store}>
           <QueryClientProvider client={queryClient}>
-            <LoadingBoundary>
+            <LoadingBoundary fallbackProps={{style: StyleSheet.absoluteFill}}>
```

Comment:
this should never trigger after the splash screen is dismissed
make fullscreen to better see it in development